bluemira/builders/EUDEMO/tools.py

```diff
@@ -280,6 +281,26 @@ def _order_shapes_anticlockwise(shapes):
     return list(np.array(shapes)[indices])
 
 
+def find_xy_plane_radii(wire, plane):
+    """
+    Get the radial coordinates of a wire's intersection points with a plane.
+
+    Parameters
+    ----------
+    wire: BluemiraWire
+        Wire to get the radii for in the plane
+    plane: BluemiraPlacement
+        Plane to slice with
+
+    Returns
+    -------
+    radii: np.ndarray
+        The array of radii of intersections, sorted from smallest to largest
+    """
+    intersections = slice_shape(wire, plane)
+    return sorted(intersections[:, 0])
```

Comment:
Won't `sorted()` return a `list`? The docstring says `ndarray`; maybe we want `np.sort` instead?